src/components/BannerRoutes.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'BannerRoutes',
+  props: {
+    routesCount: {
+      type: Number,
+      required: true,
+    },
+  },
+});
+</script>
+
+<template>
+  <q-card
```

Comment:
Do you really need a `q-card` component here instead of a simple div element? Check the `BannerImage` component, please.

Reply:
Refactored using <div>